src/realtime/commands/direct.commands.ts

```diff
@@ -185,10 +185,14 @@ export class DirectCommands {
         clientContext,
         threadId,
         reactionStatus,
+        targetItemType,
+        emoji
     }: {
         itemId: string;
         reactionType?: 'like' | string;
         reactionStatus?: 'created' | 'deleted';
+        targetItemType?: string
```

Comment:
This can only be `voice_media` or `raven_media`.
```suggestion
        targetItemType?: 'voice_media' | 'raven_media';
```

Reply:
From what I noticed it is same as thread item type. so 'text' / 'media' can also be valid. It also makes sense practically as `targetItemType` is referring to the item type of item which is the target of reaction. In instagram app, user can react to many things inside chat. But I might be wrong. I have only been using this library for ~2 hours.

Reply:
Oh then you can set it to [`MessageSyncMessageTypes`](https://github.com/Nerixyz/instagram_mqtt/blob/master/src/realtime/messages/message-sync.message.ts#L8).